plugins/woocommerce/includes/rest-api/Controllers/Version2/class-wc-rest-shipping-methods-v2-controller.php

```diff
@@ -101,14 +101,19 @@ public function get_item_permissions_check( $request ) {
 	 * @return WP_Error|WP_REST_Response
 	 */
 	public function get_items( $request ) {
+		
```

Comment:
This empty line produces a linting error. See https://github.com/woocommerce/woocommerce/actions/runs/13146532631/job/36987499029?pr=55110 for the details.

Reply:
Done